pandas/core/frame.py

```diff
@@ -7302,7 +7304,7 @@ def corr(self, method="pearson", min_periods=1):
                 will have 1 along the diagonals and will be symmetric
                 regardless of the callable's behavior.
 
-                .. versionadded:: 0.24.0
+            .. versionadded:: 0.24.0
```

Comment:
The indentation here is still incorrect I think, but it's actually also the existing lines above that are wrong. The text in that last list item should not indent more than "callable" (and then the versionadded should be at the same indentation level as this text). Now you are at it, would you like to fix this as well?